plugins/iterm2-tab-color/iterm2-tab-color.plugin.zsh

```diff
@@ -0,0 +1,43 @@
+# Written by Wade Simmons
+# https://github.com/wadey
+#
+# iTerm2 tab color commands
+# http://code.google.com/p/iterm2/wiki/ProprietaryEscapeCodes
```

Comment:
This URL is out of date. The current one is https://iterm2.com/documentation-escape-codes.html.